utils/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
```

Comment:
Does it make sense to have a single big utils crate or would it be better to have separate crates for the repl, file parser, graph utils, etc. Fine for now, just something to think about. I think the file parser specifically isn't really a "util" per se

Reply:
Yeah, it is just everything shared between the cli and web not in core